integration/system/info_test.go

```diff
@@ -65,3 +67,59 @@ func TestInfoAPIWarnings(t *testing.T) {
 		assert.Check(t, is.Contains(out, linePrefix))
 	}
 }
+
+func TestInfoDebug(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon, "cannot run daemon when remote daemon")
+
+	d := daemon.New(t)
+	d.Start(t, "--debug")
+	defer d.Stop(t)
+
+	info := d.Info(t)
+	assert.Equal(t, info.Debug, true)
+
+	// Note that the information below is not tied to debug-mode being enabled.
+	assert.Check(t, info.NFd != 0)
+
+	// TODO need a stable way to generate event listeners
+	// assert.Check(t, info.NEventsListener != 0)
+	assert.Check(t, info.NGoroutines != 0)
+	assert.Check(t, info.SystemTime != "")
+	assert.Equal(t, info.DockerRootDir, d.Root)
+}
+
+func TestInfoInsecureRegistries(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon, "cannot run daemon when remote daemon")
+
+	const (
+		registryCIDR = "192.168.1.0/24"
+		registryHost = "insecurehost.com:5000"
+	)
+
+	d := daemon.New(t)
+	d.Start(t, "--insecure-registry="+registryCIDR, "--insecure-registry="+registryHost)
+	defer d.Stop(t)
+
+	info := d.Info(t)
+	assert.Equal(t, info.RegistryConfig.InsecureRegistryCIDRs[0].String(), registryCIDR)
+	assert.Equal(t, info.RegistryConfig.InsecureRegistryCIDRs[1].String(), "127.0.0.0/8")
```

Comment:
this could potentially be flaky since `InsecureRegistryCIDRs[0]` and `[1]` can change places (as the order does not matter here I believe). If it's not happening in practice, let it be the way it is.

Reply:
to avoid flakiness, sort is required

Reply:
It's a slice, not a map, so I think they should always have the same place.

I can have a quick look to sort them